src/main/java/core/basesyntax/services/operations/Buy.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.services.operations;
+
+import java.time.LocalDate;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class Buy implements Operable {
```

Comment:
```suggestion
public class PurchaseOperation implements Operable {
```
buy is a bit abstract